ssg/build_yaml.py

```diff
@@ -232,7 +232,8 @@ def __sub__(self, other):
         profile.selected = list(set(self.selected) - set(other.selected))
         profile.selected.sort()
         profile.unselected = list(set(self.unselected) - set(other.unselected))
-        profile.variables = self.variables
+        profile.variables = {k: v for k, v in self.variables.items()
```

Comment:
This is not supported in python from RHEL6 node. Please fix it.

```
SyntaxError: invalid syntax
-- Scanning for dependencies of rhel6 fixes (bash, ansible, puppet and anaconda)...
-- Scanning for dependencies of rhel6 checks (OVAL)...
Traceback (most recent call last):
  File "/home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel6/build-scripts/yaml_to_shorthand.py", line 10, in <module>
    import ssg.build_yaml
  File "/home/jenkins/workspace/scap-security-guide-pull-requests/label/rhel6/ssg/build_yaml.py", line 235
    profile.variables = {k: v for k, v in self.variables.items()
                                ^
```

Reply:
:disappointed:

Reply:
@ggbecker Fixed